test/bin/scenario.sh

```diff
@@ -83,12 +83,31 @@ copy_file_to_vm() {
     local -r local_filename="$2"
     local -r remote_filename="$3"
 
-    local -r ip=$(get_vm_property "${vmname}" ip)
+    local ip
+    ip=$(get_vm_property "${vmname}" ip)
+    if [ "${ip}" != "${ip#*:[0-9a-fA-F]}" ]; then
+      ip="[${ip}]"
+    fi
     local -r ssh_port=$(get_vm_property "${vmname}" ssh_port)
 
     scp -P "${ssh_port}" "${local_filename}" "redhat@${ip}:${remote_filename}"
 }
 
+copy_file_from_vm() {
+    local -r vmname="$1"
+    local -r remote_filename="$2"
+    local -r local_filename="$3"
+
+    local ip
+    ip=$(get_vm_property "${vmname}" ip)
+    if [ "${ip}" != "${ip#*:[0-9a-fA-F]}" ]; then
+      ip="[${ip}]"
+    fi
+    local -r ssh_port=$(get_vm_property "${vmname}" ssh_port)
```

Comment:
It's probably not likely to fail, but it'd be good to follow the same pattern used for `ip` to make sure the exit status of the subproc is caught.

Reply:
Created https://issues.redhat.com/browse/USHIFT-3769 for this purpose. Thanks!